contribute.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Mozilla IRC iplimit",
+  "description": "Self-service tool for adding IRC exceptions for one's IP address",
+  "repository": {
+    "url": "https://github.com/mozilla/irc-iplimit",
+    "license": "MPL2"
+  },
+  "keywords": [
+    "Python",
+    "Flask",
+    "JSON"
+  ],
+	"participate": {
+		"home": "https://iplimit.irc.mozilla.org/",
+    "irc": "ircs://irc.mozilla.org",
+    "irc-contacts": "ashish"
```

Comment:
Consider making this either `#moc` or an array containing both your name and `#moc`.